svc-composition-service/app/proto/composition_service.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
```

Comment:
Please explain the reasoning behind having the proto file under the composition service. Which is an exact match of the file present under lib-utilities.